events/config.go

```diff
@@ -78,3 +78,23 @@ func MustViperFlagsForSubscriber(v *viper.Viper, flags *pflag.FlagSet) {
 
 	v.SetDefault("events.subscriber.timeout", defaultTimeout)
 }
+
+// SubscriberConfigFromViper builds a new SubscriberConfig from viper.
```

Comment:
Cleaned these up because I see where we're going with this. But we actually do have other helps like this in other `x` packages. We can clean those up as part of a separate PR though if that's not the direction we want to head.